tensorflow/core/platform/env.h

```diff
@@ -218,6 +218,10 @@ class Env {
   /// Creates a local unique temporary file name. Returns true if success.
   bool LocalTempFilename(string* filename);
 
+  /// Create a unique file name starts with |prefix| and ends with |suffix|.
```

Comment:
Nit: "Creates" (this is Google style, but also matches the surrounding code).

Reply:
Done